packages/configs/default/index.json

```diff
@@ -1,6 +1,7 @@
 {
   "bundler": "@parcel/bundler-default",
   "transforms": {
+    "types:*.{ts,tsx}": ["@parcel/transformer-typescript-types"],
```

Comment:
Is this supposed to be the syntax for named pipelines?

Reply:
Yes.